root/frontend/src/components/Weather/Weather.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import useStyles from './Weather.style';
+
+function Weather() {
+  const classes = useStyles();
+  const fetchWeatherData = async (): Promise<any> => {
+    const data = await axios('http://localhost:5005/weather/getWeatherInfo');
```

Comment:
2 things here to comment on:
1. we may want to extract `localhost:5005` to like a constants file or something for use around the program and for easier refactoring when/if we host this (though this can probably be done in the future).
2. nitpick: I think we should go with a variable name that's more descriptive than `data`